index.bs

```diff
@@ -4,14 +4,88 @@ Shortname: aria-at-automation
 Level: 1
 Status: CG-DRAFT
 Group: ARIA-AT CG
-Repository: bocoup/aria-at-automation
-URL: https://bocoup.github.io/aria-at-automation/
+Repository: w3c/aria-at-automation
+URL: https://w3c.github.io/aria-at-automation/
 Editor: Simon Pieters, Bocoup https://bocoup.com, simon@bocoup.com
 !Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/aria-at-automation>web-platform-tests aria-at-automation/</a> (<a href=https://github.com/w3c/web-platform-tests/labels/aria-at-automation>ongoing work</a>)
-Abstract: A short description of your spec, one or two sentences.
+Abstract: A short description of your spec, one or two sentences. (TODO)
 </pre>
 
 Introduction {#intro}
 =====================
 
-See https://github.com/tabatkins/bikeshed to get started.
+AT Driver defines a protocol for introspection and remote control of assistive technology software, using a bidirectional communication channel.
+
+Explainer {#explainer}
+======================
+
+Specify a protocol using WebSocket that maximally reuses concepts and conventions from WebDriver BiDi.
+
+A connection has two endpoints: remote and local. The remote end can control and read from the screen reader, which can either be implemented as a standalone application or be implemented as part of the AT software. The local end is what the test interfaces with, usually in the form of language-specific libraries providing an API.
+
+(For WebDriver, you still need to use HTTP to request the "webSocketUrl" capability, and then establish the WebSocket connection. Here we want the WebSocket connection to be the initial and only connection.)
```

Comment:
We should take care with wording here since WebSocket necessarily begins with an HTTP handshake.